frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx

```diff
@@ -0,0 +1,51 @@
+import type { HoverCardProps } from "metabase/ui";
```

Comment:
:x: 

Navigation does not happen when clicking a link to another table. Popup just disappears.
We should have a test for that case.


https://github.com/metabase/metabase/assets/6830683/57dafee3-ff81-4a1b-be82-359d5208e387

Reply:
Good catch. I had to add the `useSequencedContentCloseHandler` to the popover component and that seems to fix it.

I added a test as well.

Reply:
a11y: Clicking the link with Middle Mouse Button does not always work.
It should open the link in a new tab, but sometimes the popover gets closed instead.

https://github.com/metabase/metabase/assets/6830683/9a7f01ac-3b64-4435-bccd-d8855f660426

Reply:
I cannot reproduce this on any of my browsers (on mac): both click and option-click work as expected.

Does this work with other components that use `useSequencedContentCloseHandler`?